bin/oasp4fn

```diff
@@ -2,96 +2,9 @@
 
 'use strict';
 
-let yml_gen = require('../dist/generators/index');
-let fs = require('fs');
-let path = require('path');
-let _ = require('lodash');
-let chalk = require('chalk');
-
-const HELP = `
-Usage: oasp4fn [provider] [options]
-   or: fun [provider] [options]
-
-Supported Providers: aws (by default aws)
-
-Options:
-  -o, --opts file       file with the options for the yml generation
-  -p, --path directory  directory where the handlers are stored
-  -e, --express         generates an express app.ts file
-  -h, --help            display the help
-`
-
-function logError(err) {
```

Comment:
The error management is handled by yargs? Because keep in mind that even the errors throwed by the generetor has to be managed, not only the CLI checks.